src/OAuthClient/ClientHelper.php

```diff
@@ -50,6 +53,22 @@ public static function getResponse($url = '', $sourceFunction = '')
                     'Client Error from '. $sourceFunction . ' ' . $exception->getMessage()
                 )
             );
+        } catch (\Exception $exception) {
+            if (strpos($exception->getMessage(), 'Operation timed out') !== false) {
```

Comment:
Since you're repeating this logic, can it be refactored into one method that you can call twice?